cookbook/console/command_in_controller.rst

```diff
@@ -0,0 +1,85 @@
+.. index::
+   single: Console; Use commands in your Controller
+
+How to make use of a Command in your application
```

Comment:
I think something like "How to Call a Command from a Controller" better matches the contents.